app/controllers/api/v1/helpees_controller.rb

```diff
@@ -5,6 +5,29 @@ def index
     @helpees = Helpee.all
   end
 
+  def rating
+    @order_request = OrderRequest.find_by('order_id = ? AND order_request_status = ?', params[:order_id], OrderRequest.order_request_statuses[:accepted])
+    if !@order_request.nil?
+      @rating = HelpeeRating.new
+      @rating.order_id = @order_request.order.id
+      @rating.qualifier_id = @order_request.order.helpee.id
+      @rating.qualified_id = @order_request.volunteer.id
+      @rating.score = params[:score]
+      @rating.comment = params[:comment]
+      @rating.save!
```

Comment:
```suggestion
      rating = HelpeeRating.new
      rating.order_id = @order_request.order.id
      rating.qualifier_id = @order_request.order.helpee.id
      rating.qualified_id = @order_request.volunteer.id
      rating.score = params[:score]
      rating.comment = params[:comment]
      rating.save!
```
Si no se usa fuera de esta función debería ser una variable local y no de instancia. Por lo que vi en las views no se usa. Dejarla así puede llegar a generar problemas porque la misma variable está definida en "rating_pending".